zenodo/modules/github/templates/github/helpers.html

```diff
@@ -17,11 +17,11 @@
 ## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 #}
 
-{%- macro doi_badge(doi, modal_name='') %}
+{%- macro doi_badge(doi, user_id, repository, modal_name='') %}
```

Comment:
I think this will change the badges used on the record pages, which the code should not do. I think that only if user_id and repository is provided should generate the new URLs.